src/main/java/cz/brmlab/yodaqa/pipeline/AnswerCASMerger.java

```diff
@@ -207,12 +208,13 @@ protected Answer makeAnswer(JCas canAnswer, AnswerInfo ai, JCas hitlistCas) {
 		answer.setText(canAnswer.getDocumentText());
 		answer.setCanonText(ai.getCanonText());
 		answer.setAnswerID(ai.getAnswerID());
-		if (ai.getPassageIDs() != null) { //Answer Info passageID is null when we created it using SORLDOC or structured search
-			answer.setPassageIDs(new IntegerArray(hitlistCas, ai.getPassageIDs().size()));
-			answer.getPassageIDs().copyFromArray(ai.getPassageIDs().toArray(), 0, 0, ai.getPassageIDs().size());
+		if (ai.getSnippetIDs() != null) { //Answer Info passageID is null when we created it using SORLDOC or structured search
+											//since we now use AnsweringSnippet, this should never be null!!
+			answer.setSnippetIDs(new IntegerArray(hitlistCas, ai.getSnippetIDs().size()));
+			answer.getSnippetIDs().copyFromArray(ai.getSnippetIDs().toArray(), 0, 0, ai.getSnippetIDs().size());
 		}
 		else { //create new IntegerArray of size 0
- 			answer.setPassageIDs(new IntegerArray(hitlistCas, 0));
+		answer.setSnippetIDs(new IntegerArray(hitlistCas, 0));
```

Comment:
Style: Broken indentation.

(Also, we don't write a newline between } and else {. But that's not too important.)